recipe/bld.bat

```diff
@@ -0,0 +1,13 @@
+python -m pip install --no-deps --ignore-installed .
+
+REM Get Python sys.prefix
+FOR /F "tokens=* USEBACKQ" %%F IN (`python -c "import sys; print(sys.prefix)"`) DO (
+    SET JUPYTERLAB_DIR=%%F\share\jupyter\delux
+)
+
```

Comment:
As below, I think we could use the PREFIX conda build environment variable here, instead of the Python subprocess approach:

```
SET JUPYTERLAB_DIR=%PREFIX%\share\jupyter\delux
```

Reply:
I'll change this one too.